### DISCUSSION TITLE
How to delete a disabled repo(forked)

### DISCUSSION THREAD

**Discussion by stev3dev** [dev-protocol/community#720] [Q&A] [answered]
I can't delete a repo. I recently found out that this repository that I had forked, has been disabled. My fork was disabled too. I wanted to delete it, but I couldn't. Thank you for taking the time to read this.

**Comment by ngljcb** [comment]
try to delete it through the repos settings

**Comment by stev3dev** [comment]
Unfortunately I can't access the repo's settings. I think because it was taken down due to a DMCA Takedown Procedure.

**Comment by ngljcb** [comment, marked as answer]
If you're not able to delete it through the repository settings, you should [contact GitHub support](https://support.github.com/request) and ask them to take care of it.
